### PR TITLE
Kotlin 1.3.71 upgrade

### PR DESCRIPTION
**Pull request by yschimke**

### CODE REVIEW

**Comment by yschimke** [comment]
Nothing meaningful https://github.com/JetBrains/kotlin/releases/tag/v1.3.71

File: build.gradle [hunk 1]
```diff
@@ -204,6 +204,7 @@ subprojects { project ->
   }
 
   dependencies {
+    //noinspection GradleDynamicVersion
```
Comment (yschimke) [inline build.gradle:207]: + flagged as dynamic version

Comment (swankjesse) [reply build.gradle:207]: Which `+` ?

Comment (yschimke) [reply build.gradle:207]: com.google.errorprone:javac:9+181-r4173-1

Comment (yschimke) [reply build.gradle:207]: I can revert and raise a bug against the gradle project I guess?

**Pull request merged**